src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -301,7 +302,11 @@ private NuGetPackageManager GetNuGetPackageManager(string solutionDirectory)
             // and not have to determine if the package is a satellite package beforehand
             if (nuGetProjectContext.PackageExtractionContext == null)
             {
-                nuGetProjectContext.PackageExtractionContext = new PackageExtractionContext(new LoggerAdapter(nuGetProjectContext));
+                var signedPackageVerifier = new SignedPackageVerifier(
+                           SignatureVerificationProviderFactory.GetSignatureVerificationProviders(),
+                           SignedPackageVerifierSettings.Default);
+
+                nuGetProjectContext.PackageExtractionContext = new PackageExtractionV2Context(new LoggerAdapter(nuGetProjectContext), signedPackageVerifier: signedPackageVerifier);
```

Comment:
Remove the named parameter `signedPackageVerifier`; it's unnecessary, and the local variable is already well-named.